### PR TITLE
test: add query test scenario w/ missing columns in different chunks

### PR DESCRIPTION
**Pull request by crepererum**
I originally hoped that this would trigger a panic but it doesn't. The test is useful anyways.

### CODE REVIEW

File: query_tests/src/cases.rs [hunk 1]
```diff
@@ -134,28 +134,34 @@ async fn test_cases_two_sql() {
 // Tests from "several_chunks.sql",
 async fn test_cases_several_chunks_sql() {
     let input_path = Path::new("cases").join("in").join("several_chunks.sql");
-    let output_path = make_output_path(&input_path).unwrap();
-    let expected_path = input_path.with_extension("expected");
+    let mut runner = Runner::new();
+    runner
+        .run(input_path)
+        .await
+        .expect("test failed");
```
Comment (mkmik) [inline query_tests/src/cases.rs:141]: I'm always confused about the polarity of the expect message. Does this mean that we expected the test to fail but it didn't?

**Review by mkmik — APPROVED** [review]
Test looks good. I have one question which can be answered async

File: query_tests/src/cases.rs (hunk 1 above)
Comment (crepererum) [reply query_tests/src/cases.rs:141]: :shrug: I have the same question :smile:

File: query_tests/cases/in/two_chunks_missing_columns.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-- Basic query tests
+-- IOX_SETUP: TwoChunksMissingColumns
+
+-- query data
+SELECT * from "table" order by time;
```
Comment (NGA-TRAN) [inline query_tests/cases/in/two_chunks_missing_columns.sql:5]: I am sorry but I do not understand the purpose of the test. It seems from the name of `TwoChunksMissingColumns`, we want to test how things work with 2 missing columns but the `select * from "table" order by time;` seems not do that job. Do you mean to select specific column names that do not exist in the table or the `where` condition includes missing columns?

**Pull request merged**